Add docs about zoho flow

z4kn4fein opened the pull request.

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+You can set **triggers**, add **actions**, use **Delays and Decisions** to create your entire workflow on an easy to use builder.
+
+<a href="https://www.zoho.com/flow/apps/configcat/integrations" target="_blank">ConfigCat's Zoho Flow integration</a> can notify you about Feature Flags or Settings changes in ConfigCat, so you can configure e.g. with Slack to send a message about the changes applied in the ConfigCat environment.
```
<a href="https://www.zoho.com/flow/apps/configcat/integrations" target="_blank">ConfigCat's Zoho Flow integration</a> can notify you about Feature Flags or Settings changes in ConfigCat, so fox example you can set it up with Slack to send a message about the feature flag changes in ConfigCat.

z4kn4fein replied:
Done

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+1. Log in to your <a href="https://www.zoho.com/flow/" target="_blank">Zoho Flow account</a> or create a new one.
+
+1. Navigate to `My Flows` on the side menu bar.
```
1. Navigate to `My Flows` using the side menu.

z4kn4fein replied:
Done

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+![zoho_product_config_environment](/assets/select_product_config_environment.png)
+
+1. Once that's done you can continue to build your flow, the trigger is handled by ConfigCat!
```
`trigger` is not a word used in ConfigCat documentation, meaning needs to be explained.

z4kn4fein replied:
It's a word in the Zoho Flow's world, so I assume someone using this integration does know what a trigger is, also this setup section started with the creation of the ConfigCat app trigger.

sigewuzhere replied:
I see, so this refers to the same "trigger". Suggestion:
Your trigger is set and will be called by ConfigCat. You can go ahead building your flows. See the list of <a href="https://www.zoho.com/flow/apps/" target="_blank">available Zoho Flow Apps</a> for inspiration.

So next step can be removed.

z4kn4fein replied:
Done

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+1. Once that's done you can continue to build your flow, the trigger is handled by ConfigCat!
+
+1. Need inspiration? See how much <a href="https://www.zoho.com/flow/apps/" target="_blank">app you can connect</a> to ConfigCat.
```
Now you can connect Zoho Flow to ConfigCat. See the list of <a href="https://www.zoho.com/flow/apps/" target="_blank">available Zoho Flow Apps</a> for inspiration.

z4kn4fein replied:
Done.

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+1. Need inspiration? See how much <a href="https://www.zoho.com/flow/apps/" target="_blank">app you can connect</a> to ConfigCat.
+
+## Example Slack notification setup
```
Example on how to get Slack notifications on feature flag changes.

z4kn4fein replied:
Done.

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+## Example Slack notification setup
+
+1. Use the `Send public/private channel message` action from the Slack app and drag it under the ConfigCat trigger.
```
Missing first step:
1. On the left pane in the Builder, find and open Slack App.
plus screenshot of the left pane with opened Slack

z4kn4fein replied:
Yeah, the first screenshot was about that, just the path was wrong, could you please re-check?

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+![zoho_slack_add](/assets/add_slack.png)
+
+1. Configure the Slack action to which channel and exactly what it should send.
```
Which action is for feature flag changes?

z4kn4fein replied:
The `Send message` action we selected in the previous step

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+![zoho_slack_add](/assets/add_slack.png)
+
+1. Configure the Slack action to which channel and exactly what it should send.
+   > You may have to connect the Slack app to your Slack subscription if you didn't do it before.
```
Zoho Flow Slack App to your Slack subscription

z4kn4fein replied:
Done.

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+![zoho_slack_config](/assets/slack_configuration.png)
+
+1. Hit the `Test & Debug` button on the right top corner and wait for the results.
```
... wait for the results in your Slack workspace.
You should get a message like this:

z4kn4fein replied:
Done.

sigewuzhere commented:
- Build fails on: Multiple links to images are broken.
- Numbered list are wrongly numbered.
- Text is too small on images. Should be cropped.
- Please no dark mode when creating screenshots.

z4kn4fein commented:
1. Fixed the wrong paths.
1. Fixed the numbers.
1. Tried to crop smaller images, are they good now?
1. 😄  Sometimes I just forget that the world is white and I'm just using a dark mode extension. Coelho CSP ladies and gentlemen. Anyway, fixed those as well.

sigewuzhere commented on website/docs/integrations/zoho-flow.md:
```diff
@@ -0,0 +1,68 @@
+
+![zoho_fill_form](/assets/zoho/fill_connect_form.png)
+
+10. On the next pane you have to select the ConfigCat [Product](/main-concepts#product), [Config](/main-concepts#config) and [Environment](/main-concepts#environment) you want to be notified about. Optionally, you can set filter conditions to control exactly when should the trigger fire.
```
Maybe it's just me, but I'm missing an explanation on what am I supposed to have in the `Variable name` field.

z4kn4fein replied:
That's auto-generated and pre-filled when the pane opens. You can change its value but doesn't have much importance here

sigewuzhere replied:
Okay, cool!
Suggestion: The `Variable name` field is pre-filled by [blah-blah], you can leave it as is.

z4kn4fein replied:
Alrighty, it's done

sigewuzhere reviewed (commented):
1. Thank you
2. You da best
3. They rock 🤘 
4. The world is not just black and white my dear Coelho. 😁
